src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/person/domain/OppfolgingstilfelleDag.kt

```diff
@@ -13,11 +13,14 @@ private val log = LoggerFactory.getLogger(OppfolgingstilfelleDag::class.java)
 class OppfolgingstilfelleDag(
     val dag: LocalDate,
     val priorityOppfolgingstilfelleBit: OppfolgingstilfelleBit?,
+    val priorityGraderingBit: OppfolgingstilfelleBit?,
     val virksomhetsnummerPreferred: List<String>,
     val virksomhetsnummerAll: List<String>,
 )
 
-fun List<OppfolgingstilfelleDag>.groupOppfolgingstilfelleList(): List<Oppfolgingstilfelle> {
+fun List<OppfolgingstilfelleDag>.groupOppfolgingstilfelleList(
```

Comment:
Kan metodenavnet her si noe om hva som er kriteriet for gruppering??

Reply:
Usikker på bruken av group her så renamet bare til `toOppfolgingstilfelleList` (man går fra en liste med oppfolgingstilfelledager til en liste med oppfolgingstilfeller)